Typescript: compile time error for native textarea attributes

Hello, thanks for support Typescript out-of-box.

But i have a trouble:

> error TS2339: Property 'name' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<Component<TextareaAutosizeProps, ComponentState>> ...

from this code:

```jsx
import * as React from 'react';
import TextareaAutosize from 'react-autosize-textarea';

interface Props {
    onChange: React.EventHandler<React.ChangeEvent<HTMLTextAreaElement>>;
    value: string;
}

export default class Textarea extends React.Component<Props, {}> {
  render() {
    return (
      <TextareaAutosize
         className="textarea__control"
         name="message"
         placeholder="Your message"
         onChange={this.props.onChange}
         value={this.props.value}
      />
    );
  }
}
```
I look at `react-autosize-textarea.d.ts` and think that have an error:

```diff
/// <reference types="react" />

import * as React from 'react';

--- export interface TextareaAutosizeProps extends React.HTMLAttributes<HTMLTextAreaElement> {
+++ export interface TextareaAutosizeProps extends React.TextareaHTMLAttributes<HTMLTextAreaElement> {
  onResize?: (e: React.SyntheticEvent<Event>) => void,
  // rows is already typed in `React.HTMLAttributes<HTMLTextAreaElement>`
  maxRows?: number,
  innerRef?: (textarea: HTMLTextAreaElement) => void
}

declare const TextareaAutosize: React.ComponentClass<TextareaAutosizeProps>;
export default TextareaAutosize
```

`React.HTMLAttributes` doesn't have a `textarea` attributes, such as `name`, `placeholder`, ...

I replaced `React.HTMLAttributes` wtih `React.TextareaHTMLAttributes` and it work fine.

--- a/package.json
+++ b/package.json
@@ -44,7 +44,7 @@
   "typings": "react-autosize-textarea.d.ts",
   "homepage": "https://github.com/buildo/react-autosize-textarea",
   "devDependencies": {
-    "@types/react": "^15.0.24",
+    "@types/react": "^15.0.38",
     "babel-cli": "^6.11.4",
     "babel-core": "^6.21.0",
     "babel-loader": "^6.2.4",

--- a/react-autosize-textarea.d.ts
+++ b/react-autosize-textarea.d.ts
@@ -2,7 +2,7 @@
 
 import * as React from 'react';
 
-export interface TextareaAutosizeProps extends React.HTMLAttributes<HTMLTextAreaElement> {
+export interface TextareaAutosizeProps extends React.TextareaHTMLAttributes<HTMLTextAreaElement> {
   onResize?: (e: React.SyntheticEvent<Event>) => void,
   // rows is already typed in `React.HTMLAttributes<HTMLTextAreaElement>`
   maxRows?: number,

